x/concentrated-liquidity/types/constants.go

```diff
@@ -0,0 +1,10 @@
+package types
+
+import (
+	sdk "github.com/cosmos/cosmos-sdk/types"
+)
+
+var (
+	MaxTick = sdk.NewIntFromUint64(887272)
```

Comment:
Can we add TODO / issue to decide on the value here

Reply:
Added in https://github.com/osmosis-labs/osmosis/issues/3030 as well as comment in code